packages/xo-web/src/xo-app/logs/backup-ng/log-alert-header.js

```diff
@@ -66,6 +68,21 @@ export default decorate([
       formattedLog: (_, { log }) => JSON.stringify(log, null, 2),
       jobFailed: (_, { log = {} }) =>
         log.status !== 'success' && log.status !== 'pending',
+      files: ({ formattedLog }, { log }) =>
+        !OPEN_SOURCE_USER
+          ? [
+              {
+                file: new window.Blob([formattedLog], {
+                  type: 'application/json',
+                }),
+                name: `${new Date(log.start)
+                  .toISOString()
+                  .replace(/:/g, '_')} - backup NG.log`,
+              },
+            ]
+          : undefined,
+      message: ({ formattedLog }) =>
+        OPEN_SOURCE_USER ? `\`\`\`json\n${formattedLog}\n\`\`\`` : undefined,
```

Comment:
Can `OPEN_SOURCE_USER`  send file when repoting Bug? 
Can other users send message ?
We can add for other users to send message and file ?

Reply:
This behavior is done automatically. The user don't send a file or put a message in XO, we do this things for him. He can do it on Github(open source user)/Gitlab(user with plan).